### PR TITLE
Fix failure if web3signer isn't running

### PR DESCRIPTION
**Pull request by courtneyeh**
#7422 

## Documentation

- [X] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [X] I thought about adding a changelog entry, and added one if I deemed necessary.

### CODE REVIEW

File: validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java [hunk 1]
```diff
@@ -243,6 +245,17 @@ public static ValidatorClientService create(
                   .propagateTo(validatorClientService.initializationComplete);
               return SafeFuture.COMPLETE;
             })
+        .exceptionally(
+            er -> {
+              Optional<Throwable> maybeCause =
```
Comment (rolfyone) [inline validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java]: final, and lets call 'er' something else? `error` and `err` seem to be common, i think `error` is better...

File: validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java [hunk 2]
```diff
@@ -243,6 +245,17 @@ public static ValidatorClientService create(
...
+                  ExceptionUtil.getCause(er, InvalidConfigurationException.class);
+              if (maybeCause.isPresent()) {
+                LOG.warn(maybeCause.get().getMessage());
+              } else {
+                LOG.error("Error was encountered during validator client service start up.", er);
+              }
+              return null;
+            })
         .ifExceptionGetsHereRaiseABug();
```
Comment (rolfyone) [inline validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java]: this cant stay here iirc, we need to remove `ifExceptionGetsHereRaiseABug` - we've handled the error before that...

Comment (courtneyeh) [reply validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java]: Yes just working that one out. The compiler complains if we don't check the result of the `exceptionally()`.

Comment (rolfyone) [reply validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java]: can probably just do a .always wtih log.trace 'finished starting validator client service'

File: validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java [hunk 3]
```diff
@@ -243,7 +245,18 @@ public static ValidatorClientService create(
                   .propagateTo(validatorClientService.initializationComplete);
               return SafeFuture.COMPLETE;
             })
-        .ifExceptionGetsHereRaiseABug();
+        .exceptionally(
+            error -> {
+              Optional<Throwable> maybeCause =
```
Comment (rolfyone) [inline validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java]: final...

**Review by rolfyone — APPROVED** [review]
LGTM, one nit.

**Pull request merged**